src/scss/orejime-dark.scss

```diff
@@ -2,13 +2,13 @@ $orejime-box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.2), 5px 5px 10px 0 rgba(0, 0, 0
 $orejime-border-radius: 0 !default;
```

Comment:
@laurent-b4y just need the orejime-light now, you can add it to this branch since its introducing the distinction.